### DISCUSSION TITLE
Bold formatting not resetting in lore

### DISCUSSION THREAD

**Discussion by Dani3l63** [NichtStudioCode/InvUI#83] [Q&A] [answered]
Hello, i want to report a problem with resetting the bold text regardless of whether i use the legacy mode or not, as stated in a previous issue created by another user. #66 

If i'm doing something wrong, i apologize for wasting your time.

![image](https://github.com/user-attachments/assets/93d63bdb-7a2d-458c-a536-06c102975c2b)
![image](https://github.com/user-attachments/assets/84b89440-edcb-4d72-8173-d3916d680277)

**Comment by NichtStudioCode** [comment, marked as answer]
You're using the legacy format. Use BaseComponent[] or, even better, [kyori adventure text components](https://docs.advntr.dev/text.html) (built into Paper) via ComponentWrapper.
(Also `§f` isn't a reset code)
